src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +210,46 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKeys = useMemo(() => [], []);
+  const excludeKeys = useMemo(() => ['calls'], []);
+
+  const loadWorkflows = useCallback(
+    async (workflowId: string | undefined): Promise<WorkflowMetadata | undefined> => {
```

Comment:
Nit: the name here is confusing, since it suggests that it's loading multiple workflows, but it looks like it only loads one specified workflow?